AmericanMedicalAssociation/ama-style-guide-2
# EWL-10882: Update article listicle styling.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

## JIRA Ticket(s)
- [EWL-10882: Article Template | Update Listicle Style](https://issues.ama-assn.org/browse/EWL-10882)


## Description:
Add new listicle styling options and update listicle styling on articles.


## To Test:
- import latest config
- checkout and link ama-d8 PR: https://github.com/AmericanMedicalAssociation/ama-d8/pull/4928
- clear caches
- navigate to a listicle article, confirm styling matches the following zeplins:
Numbers:
Desktop: https://zpl.io/NEwo1yv
Tablet: https://zpl.io/8ElNJx6
Mobile: https://zpl.io/MLdv1Zz

- Edit any article, create a bulleted ordered list using the WYSIWYG in the text body
- Note make sure the lists are comprised of heading 2 or 3 with anchor tags within
- highlight the whole list and select the style dropdown, select "Bulleted List Heading"
- confirm styling matches the following zeplins
Bullets:
Desktop: https://zpl.io/AOmEn8p
Tablet: https://zpl.io/Zq0zm5J
Mobile: https://zpl.io/8ElNRn4

- Edit any article, create a bulleted ordered list using the WYSIWYG in the text body
- Note make sure the lists are comprised of heading 2 or 3 with anchor tags within
- highlight the whole list and select the style dropdown, select "Unmarked List Heading"
- confirm styling matches the following zeplins
No Markers:
Desktop: https://zpl.io/zw8nL0X
Tablet: https://zpl.io/jpZw7qW
Mobile: https://zpl.io/p01B7Rj

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.


## Additional Notes
Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
